### PR TITLE
Additional updates to walkthroughs

### PR DESCRIPTION
**Pull request by tiffanynolan**
Some additional changes to the SPs to make tasks more standard and fixed a few typos.

### CODE REVIEW

File: walkthroughs/2-protecting-apps-sso/walkthrough.adoc [hunk 1]
```diff
@@ -108,7 +108,7 @@ In the *Settings* tab on the *Client* page, make sure the *Enabled* toggle is se
 
 To secure your application with single sign-on and allow additional users to log in to your instance, you need to create users in a realm.
 
-Follow these steps to create a user.
+Follow these steps to To create a new user and set that user's password.
```
Comment (evanshortiss) [inline walkthroughs/2-protecting-apps-sso/walkthrough.adoc]: `To` vs. `to` typo?

File: walkthroughs/3-low-code-api-development/walkthrough.adoc [hunk 2]
```diff
@@ -129,41 +129,42 @@ Follow these steps to create a *Path*. A *Path* represents an API endpoint and t
 . Click *Add* to confirm your entry.
 
 . Click your new */greeting* endpoint under the *Paths* heading.
-
-. Click the *Create Operation* button for *POST* in the *Operations* section on the right. The *POST* item will become an orange button.
-
-. Click the orange *POST* button to access the *Design* screen for the *POST* operation.
-
-. Enter the following in the now visible *Summary* field:
+. In the *Operations* section on the right, select the *Post* tab.
```
Comment (evanshortiss) [inline walkthroughs/3-low-code-api-development/walkthrough.adoc:132]: Should POST be uppercase? Might be changed in a newer API Studio version I've not used, so just checking.

File: walkthroughs/3-low-code-api-development/walkthrough.adoc [hunk 3]
```diff
@@ -129,41 +129,42 @@ Follow these steps to create a *Path*. A *Path* represents an API endpoint and t
...
+. In the *RESPONSES* section, select the *Add a response* link.
 .. In the window that appears choose *200 OK* as the option and click the *Add* button.
 .. Click *No description* beside the newly listed *200 OK* response, and enter the following in the *Description* field:
 +
 ----
 Greeting response
 ----
-.. Click the *No Type* field and change the *Type* to *String as String*.
+.. For the *Response Type*, click *Choose Type* and then select *String*.
+.. You will then see an additional drop-down displayed, with the *Type* now showing "*String* as *String*".
```
Comment (evanshortiss) [inline walkthroughs/3-low-code-api-development/walkthrough.adoc]: Should this be on the same step as above since it's not an actionable step on its own?

**Review by evanshortiss — APPROVED** [review]
LGTM, just some minor questions that might not need addressing.

File: walkthroughs/3-low-code-api-development/walkthrough.adoc (hunk 2 above)
Comment (tiffanynolan) [reply walkthroughs/3-low-code-api-development/walkthrough.adoc:132]: Yes, it is uppercase in the latest version of API Designer.

**Pull request merged**